index.php

```diff
@@ -1,37 +1,108 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <link href="conference.css" type="text/css" rel="stylesheet" />
-  </head>
-  <body>
-    <nav>
-      <ul class="navbar">
-        <li><a class="active" href="index.php">Events</a></li>
-        <li><a href="attendees.php">Attendees</a></li>
-        <li><a href="sponsors.php">Sponsors</a></li>
-        <li><a href="jobs.php">Jobs</a></li>
-        <li><a href="hotel.php">Hotel</a></li>
-        <li><a href="committees.php">Committees</a></li>
-        <li><a href="logistics.php">Logistics</a></li>
-      </ul>
-    </nav>
-    <header>
-      <!-- placeholder -->
-      <h1>stuff for events</h1>
+    <head>
+        <link href="conference.css" type="text/css" rel="stylesheet" />
+    </head>
+    <body>
+        <nav>
+            <ul class="navbar">
+                <li><a class="active" href="index.php">Events</a></li>
+                <li><a href="attendees.php">Attendees</a></li>
+                <li><a href="sponsors.php">Sponsors</a></li>
+                <li><a href="jobs.php">Jobs</a></li>
+                <li><a href="hotel.php">Hotel</a></li>
+                <li><a href="committees.php">Committees</a></li>
+                <li><a href="logistics.php">Logistics</a></li>
+            </ul>
+        </nav>
+        <header>
+            <h1>Session Events</h1>
+        </header>
+        <div class="main">
+            <?php include 'pdo.php'?>
 
-    </header>
-    <div class="main">
-      <h2>list all events for a particular day</h2>
-      <h2>switch a session's day/time and/or location</h2>
-      <!-- placeholder -->
-      <?php
+            <div id="listDayEvents">
+                <h2>Event Date:</h2>
+                <select id='dateselect' onChange='displayEvent(this.value)'>
```

Comment:
on refresh, if the selection was on Saturday before, there's no call being made so we the table turns up empty. i've mitigated a similar situation in attendees so take a look there.